e2e/.env.example

```
@@ -1,2 +1,6 @@
 E2E_ACCOUNT_EMAIL=example
 E2E_ACCOUNT_PASSWORD=example
+# Debug failed tests (keeps containers)
+PRESERVE_ENV=false
+# recommended to leave blank
+GHOST_IMAGE_TAG=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GHOST_IMAGE_TAG key should go before the PRESERVE_ENV key

(UnorderedKey)